src/main/frontend/components/file.cljs

```diff
@@ -73,11 +74,12 @@
   (let [path (get-path state)
         format (format/get-format path)
         page (db/get-file-page path)
+        random-id (str (dc/squuid))
         config? (= path (config/get-config-path))]
     (rum/with-context [[tongue] i18n/*tongue-context*]
-      [:div.file {:id (str "file-" path)}
+      [:div.file {:id (str "file-edit-wrapper-" random-id)}
        [:h1.title
-        path]
+        [:bdi path]]
```

Comment:
With the following:
```css
.file h1.title {
  font-family: monospace;
  font-weight: normal;

  max-width: 100%;

  text-overflow: ellipsis;
  white-space: nowrap;
  overflow: hidden;
  text-align: left;
  direction: rtl;
}
```

![image](https://user-images.githubusercontent.com/72891/146665225-0459eb30-8af5-4dbc-85d6-f01664375fef.png)